src/main/kotlin/org/example/springaipractice/service/AdvancedAIService.kt

```
@@ -0,0 +1,82 @@
+        return response.result.output.content
+    }
+    
+    // 감정 분석 기능
+    suspend fun analyzeSentiment(text: String): SentimentResponse {
+        val prompt = promptTemplateService.createSentimentAnalysisPrompt(text)
+        val response: ChatResponse = anthropicChatClient.call(prompt)
+        val jsonResponse = response.result.output.content
+        
+        return try {
+            objectMapper.readValue(jsonResponse, SentimentResponse::class.java)
+        } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)